### PR TITLE
修改README编译部分的笔误

### PR DESCRIPTION
**Pull request by geniusLcy**
1. 修改README编译部分的笔误：
`bibtex tjumain.tex` 应该为 `bibtex tjumain.aux` 或者直接为 `bibtex tjumain` 即可，当时粗心了，抱歉
2. README示例代码加上缩进，结构稍微好看一些

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,10 +121,10 @@ $D$(in) & $P_u$(lbs) & $u_u$(in) & $\beta$ & $G_f$(psi.in)\\
 ## 编译
 
 ```bash
-xelatex tjumain.tex
-bibtex tjumain.tex
-xelatex tjumain.tex
-xelatex tjumain.tex
+xelatex tjumain
```
Comment (xnth97) [inline README.md:124]: 建议保留扩展名，或者能不能使用 `xelatex tjumain.tex && bibtex tjumain.aux && xelatex tjumain.tex && xelatex tjumain.tex` 便于用户直接 copy

Comment (geniusLcy) [reply README.md:124]: 感谢review!

### 扩展名
同意保留扩展名

### 单命令编译
不可以使用 `xelatex tjumain.tex && bibtex tjumain.aux && xelatex tjumain.tex && xelatex tjumain.tex` 在单行完成命令，但是可以使用 `latexmk` 完成简洁的单行编译命令（如果用户有latexmk的话，这玩意应该整latex环境的时候都默认都装上了）。马上更新README的编译部分内容。

**Comment by geniusLcy** [comment]
现在直接copy用latexmk的命令就可以，比以前方便一些。

File: README.md [hunk 2]
```diff
@@ -120,15 +120,90 @@ $D$(in) & $P_u$(lbs) & $u_u$(in) & $\beta$ & $G_f$(psi.in)\\
 
 ## 编译
 
+> - 以下编译操作都**需要用户进入 `tjumain.tex` 所在目录下**
+> - 有多种编译形式，本质都一样，选择某种自己喜欢的即可，欢迎补充。
+
+### 使用`Latexmk`编译
+
+#### 首先检查是否安装了`Latexmk`
```
Comment (xnth97) [inline README.md:128]: Checking installation and explaning working mechanism don't seem to be a template's scope. We should only provide minmal and proper commands to unblock users from using the template. These parts would be fit for a latex 101 guide for TJU students

File: README.md [hunk 3]
```diff
@@ -120,15 +120,90 @@ $D$(in) & $P_u$(lbs) & $u_u$(in) & $\beta$ & $G_f$(psi.in)\\
...
+运行命令，`Latexmk`会使用 xelatex 引擎编译 `tjumain.tex` 文件，并在 PDF 阅读器中打开预览，并持续更新文件。当然，它也会监测文件保存动作，并自动重新编译。
+
+使用`Latexmk`应该可以做到只需运行一次，然后每次文件保存动作后，自动重新编译。
+
+#### 清理临时文件
+
+运行LaTeX之后，当前目录被大量临时文件污染；可以使用这个命令清理
+
+```bash
+latexmk -c
```
Comment (xnth97) [inline README.md:178]: Can add a section under 清理缓存及日志

File: README.md [hunk 4]
```diff
@@ -120,15 +120,90 @@ $D$(in) & $P_u$(lbs) & $u_u$(in) & $\beta$ & $G_f$(psi.in)\\
...
 xelatex tjumain.tex
-bibtex tjumain.tex
+bibtex tjumain.aux
 xelatex tjumain.tex
 xelatex tjumain.tex
 ```
 
 注意：由于存在目录、参考文献和图表编号等，需要多次编译以保证顺序正确。
 
+### 其他编译方式
```
Comment (xnth97) [inline README.md:202]: Let's not include things we don't have

**Review by xnth97 — COMMENTED** [review]
sorry i don't have chinese IME on this computer

thanks for contribution. these contents are great but i feel its kinda off the topic. usually a README doc should provide minimal info for unblocking users instead of a detailed instruction. `latexmk` is only an optional tool and putting too much content here can be distracting for beginners

**Comment by geniusLcy** [comment]
赞成

**Review by xnth97 — APPROVED** [review]

**Comment by xnth97** [comment]
RB merged. Thanks!

**Pull request merged**